.travis.yml

```diff
@@ -19,6 +22,12 @@ matrix:
       env: DB=mysql DJANGO=1.7
     - python: "3.5"
       env: DB=postgres DJANGO=1.7
+    - python: "3.3"
+      env: DB=sqlite3 DJANGO=1.9
+    - python: "3.3"
```

Comment:
Oh, I misread this section, my bad. It's exclusions and I was reading it as inclusions. Sorry.